libs/lawnchair/lawnchairMemoryAdapter.js

```diff
@@ -1,105 +1,107 @@
-Lawnchair.adapter('memory', (function(){
+module.exports = function (Lawnchair) {
```

Comment:
Formatting got a little wild here. Should we run add standard and use `standard --fix` or similar to keep things consistent? Maybe an `.editorconfig` too?

Reply:
yeah, or we could move them out into its own module. It shouldn't be in this repo I think.

Reply:
I think these should reside in `require('lawnchair').memoryAdapter`.

I'm not sure where the code for these adapters come from originally, since I couldn't find them standalone on npm or on the original repository for lawnchair...

For styling I think we might be a litle closer to `semi-standard` than `standard`, just because semicolons are a hot topic. We have a recommended set of styles for our own code somewhere but I couldn't find it right now, going to look further.

Reply:
> I'm not sure where the code for these adapters come from originally, since I couldn't find them standalone on npm or on the original repository for lawnchair...

They were renamed, removed etc. long time ago. 
We using some really old ones. Here is tag that has some of them:
https://github.com/brianleroux/lawnchair/tree/0.6.4/src/adapters